### ISSUE TITLE
CPython 3.11 wheels are not available on PyPI

### ISSUE THREAD

**Issue by altendky** [aio-libs/frozenlist#342]
## Long story short

While it appears based on a couple other issues that frozenlist has been updated to work with 3.11 in version 1.3.1, no 3.11 wheels were uploaded to PyPI.  https://pypi.org/project/frozenlist/1.3.1/#files  I understand that at the time of that release the 3.11 interfaces may not have been finalized so there may have been good reason not to publish them at that time.

## Expected behaviour

Wheels are available for CPython 3.11 on PyPI.

## Actual behaviour

Wheels are not available and must be built locally, even if this is done automatically.

## Steps to reproduce

Look at https://pypi.org/project/frozenlist/1.3.1/#files and search for `cp311` and note no search hits.

## Your environment

frozenlist is a transitive dependency that is not pinned for us and we operate on all platforms, Intel and ARM, and CPython 3.7 through 3.11.

**Comment by Dreamsorcerer** [comment]
@webknjaz Any ideas? Is it because it was built with prerelease=False here?
https://github.com/aio-libs/frozenlist/actions/runs/2785213955/jobs/4384187464#step:8:58

**Comment by thatrandomperson5** [comment]
Any updates on a release? I'm trying to install aiohttp

**Comment by CringleySDays** [comment]
I presume there's no ETA until frozenlist updates wheels to sync with 3.11 changes?

**Comment by dnnsthnnr** [comment]
Seems like this is the case due to the CI workflow not running successfully: https://github.com/aio-libs/frozenlist/actions/runs/3391664330/jobs/5637039836 
Which fails due to the usage of a deprecated GH command (set-output) in [.github/workflows/ci.yml:94:12](https://github.com/aio-libs/frozenlist/blob/master/.github/workflows/ci.yml#L95)

**Comment by Dreamsorcerer** [comment]
Taking a look at the original issue a little closer, it appears that the default behaviour is to build for release candidates, and that builds for beta releases should not be deployed: https://cibuildwheel.readthedocs.io/en/stable/options/#prerelease-pythons

The timing for the previous release was 6 days before the release candidate, so we just missed it. Will have to pay attention to that in future.

**Comment by webknjaz** [comment]
Yep, I didn't want to release the wheels before the final release. And post that date, dependabot updating cibuildwheel enabled the Cython version automatically.

Thanks for taking care of this!
